src/App.tsx

```diff
@@ -1,8 +1,104 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import TodoList from './components/TodoList';
+import { Todo, User } from './interfaces';
+import './App.scss';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [users, setUsers] = useState<User[]>([]);
+  const [show, setShow] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(false);
```

Comment:
```suggestion
  const [show, setShow] = useState(false);
  const [loading, setLoading] = useState(false);
```
тут не обязательно указывать тип, значение будет браться с указанного значения